tests/affine-local/e2e/local-first-collections-items.spec.ts

```diff
@@ -100,13 +98,10 @@ test('edit collection', async ({ page }) => {
   const first = items.first();
   await first.hover();
   await first.getByTestId('collection-options').click();
-  const editCollection = page
-    .getByTestId('collection-option')
-    .getByText('Rename');
+  const editCollection = page.getByText('Rename');
   await editCollection.click();
-  const title = page.getByTestId('input-collection-title');
-  await title.fill('123');
-  await page.getByTestId('save-collection').click();
+  await page.keyboard.type('123', { delay: 50 });
```

Comment:
why replace fill with type + delay?

Reply:
seems [playwright docs marks `type` deprecated and recommends using `pressSequentially`](https://arc.net/l/quote/xsnpojiu)